app-backend/database/src/main/scala/com/azavea/rf/database/tables/Uploads.scala

```diff
@@ -47,6 +42,97 @@ class Uploads(_tableTag: Tag) extends Table[Upload](_tableTag, "uploads")
   val visibility: Rep[Visibility] = column[Visibility]("visibility")
```

Comment:
If we're not going to use the metadata column for this type of thing we should probably remove it to avoid future confusion. We can add it back later if it becomes useful.